cdap-app-templates/cdap-etl/pom.xml

```diff
@@ -282,6 +282,21 @@
       <version>${project.version}</version>
       <scope>test</scope>
     </dependency>
+    <dependency>
+      <groupId>com.github.wnameless</groupId>
```

Comment:
It is better we can get rid of these dependencies, why will we need to flatten the json

Reply:
because right now the pushdown config is deconstructed into smaller items into runtimeargs. @albertshau suggested that we shouldn't put giant json blogs into runtimeargs. So I need to unflatten all of these at the backend to get the original pushdown blob. I'm not sure if there's better ways to do it
![image](https://user-images.githubusercontent.com/98125204/195428955-02e6b418-75c0-4ed0-9de2-ba7597dde55e.png)

Reply:
talked with Albert offline, will make changes